html-media-capture/idlharness.window.js

```diff
@@ -0,0 +1,20 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://w3c.github.io/html-media-capture/
+
+idl_test(
+  ['html-media-capture'],
+  ['html', 'dom'],
+  idl_array => {
+    const input = document.createElement('input');
+    input.type = 'file';
+    input.accept = 'image/*';
+    input.capture = 'user';
```

Comment:
This line will mask if the capture attribute is not there at all I think, or at least lead to failures that are misleading? A plain new input element with nothing changed should suffice here.

Reply:
Done.